src/lib/crypto/kimchi_backend/common/plonk_constraint_system.ml

```diff
@@ -1899,6 +1899,11 @@ end = struct
         let lookup3 = (red_w0, (red_w5, red_w6)) in
         (* We populate with the first lookup. In the case the user uses the same
            index multiple times, the last value will be used *)
+        (* @volhovm: perhaps it would be better if runtime_lookups_rev was a set
```

Comment:
I would prefer not to include this.

Reply:
Would creating an issue with this task description be a better option?

Reply:
Created https://github.com/MinaProtocol/mina/issues/14749